src/components/MaterialUI/Surfaces/ExpansionPanel.js

```diff
@@ -0,0 +1,128 @@
+import React from "react";
+import ExpansionPanelMui from "@material-ui/core/ExpansionPanel";
+import ExpansionPanelSummary from "@material-ui/core/ExpansionPanelSummary";
+import ExpansionPanelDetails from "@material-ui/core/ExpansionPanelDetails";
+import ExpansionPanelActions from "@material-ui/core/ExpansionPanelActions";
+import { makeStyles } from "@material-ui/core/styles";
+import classNames from "classnames";
+import Icon from "./../../Icon";
+import { ExpansionPanelProps, ExpansionPanelActionsProps } from "./expansionPanelProps";
+
+const useStyles = makeStyles(theme => ({
```

Comment:
Is this the best place to put the themes for each control?  I am not saying this is bad... this means if a theme change is to be done, we will be targeting each of the wrappers.

Reply:
@TONYFECTEAU 
It's not control it's an ability to use material theme in your styles. so, if you need for example to get color from MUI theme you can access it using that parameter like color: theme.palette.primary.dark. 

theme should have same structure (designed by MUI) so, we can be sure that theme.palette.primary.dark will exist anyway.